platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -314,7 +346,12 @@ class HangingProtocolService {
 
       highestStudyMatchingScore = studyMatchDetails.score;
 
+      // TODO - use display sets instead of series, as that will use the
+      // correct re-grouped information.
```

Comment:
This is probably another thing to fix - it is quite important to use display sets, otherwise you will never see anything except the first split of a series.